### PR TITLE
[chore] runtime split tests

### PR DESCRIPTION
**Pull request by drahnr**
Move all test code to test files.

Allows for 
* [x] easier `rg` and search/inclusion/exclusion
* [x] better reviews for the sake of smaller files and changesets

Includes changes of #4821

### CODE REVIEW

**Review by sandreim — APPROVED** [review]
LGTM. Can we also do the same for `ump` which is a big file atm ?

**Review by vstakhov — APPROVED** [review]
LGTM as well

File: runtime/parachains/src/disputes/tests.rs [hunk 1]
```diff
@@ -0,0 +1,2482 @@
...
+#[test]
+fn test_import_prev_participant_spam_dec_confirmed() {
+	let mut importer = DisputeStateImporter::new(
+		DisputeState {
+			validators_for: bitvec![BitOrderLsb0, u8; 1, 0, 0, 0, 0, 0, 0, 0],
+			validators_against: bitvec![BitOrderLsb0, u8; 0, 1, 0, 0, 0, 0, 0, 0],
+			start: 0,
+			concluded_at: None,
+		},
+		0,
```
[the remark this answers is not in the document]
Comment (drahnr) [reply runtime/parachains/src/disputes/tests.rs:189]: Yep, but in a follow up please :)

File: runtime/parachains/src/disputes/tests.rs [hunk 2]
```diff
@@ -0,0 +1,2482 @@
...
+	let v = vec![1, 2, 3, 4];
+	assert_eq!(false, contains_duplicates_in_sorted_iter(&v, |a, b| a == b));
+}
+
+#[test]
+fn test_dispute_state_flag_from_state() {
+	assert_eq!(
+		DisputeStateFlags::from_state(&DisputeState {
+			validators_for: bitvec![BitOrderLsb0, u8; 0, 0, 0, 0, 0, 0, 0, 0],
+			validators_against: bitvec![BitOrderLsb0, u8; 0, 0, 0, 0, 0, 0, 0, 0],
```
[the remark this answers is not in the document]
Comment (drahnr) [reply runtime/parachains/src/disputes/tests.rs:109]: We just go with whatever the CI's `cargo +nightly fmt`  does.

**Comment by drahnr** [comment]
bot merge

**Comment by drahnr** [comment]
bot merge

**Pull request merged**